src/store/wallet.js

```diff
@@ -469,7 +485,14 @@ export default {
         }
       }
     },
-
+    getWalletIndex: function(context) {
+      const walletId = context.state.walletId
+      const walletInfos = context.state.walletInfos
+      if (walletInfos) {
+        const index = walletInfos.findIndex(wallet => wallet.id === walletId)
+        context.commit('setWalletIndex', { walletIndex: index })
+      }
+    },
```

Comment:
Is sort of strange to me that we need to look for the index of the wallet at this point when allegedly we already knew about it while in the unlock screen. Can't we set it from there when calling `unlockWallet`?